Detectors/ITSMFT/MFT/reconstruction/src/devices/FileSink.cxx

```diff
@@ -101,7 +101,7 @@ void FileSink::Init()
   
   for ( mNObjects = 0 ; mNObjects < mBranchNames.size() ; mNObjects++ ) {
 
-    LOG(INFO) << "FileSink::Init >>>>> Creating output branch \"" << mClassNames[mNObjects] << "\" with name \"" << mBranchNames[mNObjects] << "\"";
+    LOG(INFO) << R"(FileSink::Init >>>>> Creating output branch ")" << mClassNames[mNObjects] << R"(" with name ")" << mBranchNames[mNObjects] << R"(")";
 
```

Comment:
https://clang.llvm.org/extra/clang-tidy/checks/modernize-raw-string-literal.html for the full description.